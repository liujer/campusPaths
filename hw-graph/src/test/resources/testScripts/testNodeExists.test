# Tests the nodeExists method in DLGraph by adding nodes to a
# graph and determining if a particular node exists in the graph
CreateGraph A
AddNode A one
AddNode A two
AddNode A three
NodeExists A one
NodeExists A two
NodeExists A three
NodeExists A four
NodeExists A Three