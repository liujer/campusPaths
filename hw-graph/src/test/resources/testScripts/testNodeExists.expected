# Tests the nodeExists method in DLGraph by adding nodes to a
# graph and determining if a particular node exists in the graph
Created graph A
Added node one to A
Added node two to A
Added node three to A
one exists in A
two exists in A
three exists in A
four does not exist in A
Three does not exist in A